src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -22,7 +22,7 @@
             return files;
         }
 
-        if (LinkedFile.isOnlineLink(value.trim())) {
+        if (!value.trim().startsWith(":") && LinkedFile.isOnlineLink(value.trim())) {
```

Comment:
I think, `isOnlineLink` should be fixed. The file field can contain a description. In case the description is `This is an awesome/somesome file`, the method will still break.

Maybe, we should work on a more intuitive file field - see https://github.com/JabRef/jabref/issues/98 for a discussion.

Reply:
This method is only handling the edge case for non valid file links See #7948  for the PR and #7882 for the original issue

Reply:
See also the tests

Reply:
I switched to a regex with the condition to only match at the beginning. The contains www was a culprit

Reply:
I believe this was originally chose for performance reasons. Can we somehow benchmark this?

Reply:
The regex is compiled, I doubt it will be a huge problem.

Reply:
Can we remove the `!value.trim().startsWith(":")?  part?

It should be working without that check.